packages/core/src/base/staticClient.ts

```diff
@@ -109,17 +88,10 @@ export const logResponse = (response: Response): void => {
   const successString =
     response.ok || response.status === 304 ? "successful" : "unsuccessful";
   const msg = `Response: ${successString} ${response.status} ${response.statusText}`;
-  if (sdkLogger.getLevel() <= sdkLogger.levels.DEBUG) {
-    sdkLogger.debug(
-      `${msg}\nResponse Headers: ${JSON.stringify(
-        response.headers.raw(),
-        null,
-        2
-      )}`
-    );
-  } else if (sdkLogger.getLevel() <= sdkLogger.levels.INFO) {
-    sdkLogger.info(msg);
-  }
+  sdkLogger.info(msg);
```

Comment:
This is fine. The reason I had it as one message (by concatenating) is - we/sdk users don't have to try to link the messages (response with its headers in this case) when there are messages that are logged asynchronously

Reply:
Totally agree...but we should do this with a request id or a log id that traces a path throw the system.